include/core/parameters.h

```diff
@@ -875,16 +914,28 @@ namespace Parameters
     // Refinement after frequency iter
     unsigned int frequency;
 
-    // Refinement fraction
-    double refinement_fraction;
-
-    // Coarsening fraction
-    double coarsening_fraction;
-
     static void
     declare_parameters(ParameterHandler &prm);
     void
     parse_parameters(ParameterHandler &prm);
+
+    // @brief Splitting method, used to fill the map 'variables'
+    // @input variables, string with mesh adaptation variables separated by a
+    // comma
+    std::vector<std::string>
```

Comment:
You don't need a function for this. There is already one in deal.II
https://github.com/lethe-cfd/lethe/blob/master/source/core/parameters_lagrangian.cc#L362